### PR TITLE
Vendor containers/common main

### PR DESCRIPTION
**Pull request by umohnani8**
Pick up changes to move the platform code to its own directory.

#### What type of PR is this?

/kind other

#### What this PR does / why we need it:
Bumping the containers/common vendor to pick up some code move changes with the platform code

#### How to verify it
Buildah should build successfully 

#### Which issue(s) this PR fixes:
Needed for the podman farm build PR

#### Does this PR introduce a user-facing change?

```release-note
Update containers/common vendor
```

### CODE REVIEW

**Comment by umohnani8** [comment]
Once https://github.com/containers/common/pull/1693 is merged, will remove the replace in go.mod

File: internal/util/util.go [hunk 1]
```diff
@@ -41,7 +42,7 @@ func LookupImage(ctx *types.SystemContext, store storage.Store, image string) (*
 // Wrapper around libimage.NormalizePlatform to return and consume
 // v1.Platform instead of independent os, arch and variant.
 func NormalizePlatform(platform v1.Platform) v1.Platform {
-	os, arch, variant := libimage.NormalizePlatform(platform.OS, platform.Architecture, platform.Variant)
+	os, arch, variant := lplatform.NormalizePlatform(platform.OS, platform.Architecture, platform.Variant)
```
Comment (rhatdan) [inline internal/util/util.go:45]: Needs new name.

Comment (umohnani8) [reply internal/util/util.go]: yup, fixed.

**Comment by umohnani8** [comment]
Updated to vendor containers/common main. This will be ready to merge once tests are green

**Comment by nalind** [comment]
Code that calls `libimage.NormalizePlatform()` is just going to _break_ unless it's modified?

**Comment by nalind** [comment]
Opened https://github.com/containers/common/pull/1696.

**Comment by vrothberg** [comment]
> Code that calls `libimage.NormalizePlatform()` is just going to _break_ unless it's modified?

This happens ~frequently.  As long as callers are updated immediately, I think it's OK.

**Comment by umohnani8** [comment]
This is ready to be merged, can I get a final lgtm here please @nalind @rhatdan

**Comment by rhatdan** [comment]
LGTM but tests are not happy

**Pull request merged**